### DISCUSSION TITLE
graphql-java version numbering

### DISCUSSION THREAD

**Discussion by limavadakara** [graphql-java/graphql-java#3187] [General]
Is version 230521-nf-execution of the graphql-java library meant to appear as the latest version in [Maven](https://mvnrepository.com/artifact/com.graphql-java/graphql-java)? 

There isn't much difference in labelling/tagging between versions [230521-nf-execution](https://mvnrepository.com/artifact/com.graphql-java/graphql-java/230521-nf-execution) and [20.2](https://mvnrepository.com/artifact/com.graphql-java/graphql-java/20.2) in Maven but the date associated with 230521-nf-execution is May 23, 2021; so feels like an old version.

I ask as our dependency tooling ([Renovate](https://github.com/renovatebot/renovate)) is picking up version 230521-nf-execution from Maven instead of 20.2 (which appears to be the [latest stable release](https://github.com/graphql-java/graphql-java/releases/tag/v20.2) here in GitHub)

**Comment by bbakerman** [comment]
I am not sure how Maven decide that entries go on the top of that list?  Alphanumeric sorting?

We should probably look into deleting that version (if we can - not sure on the rules in Maven of deleting things)

**Reply by dondonz** [reply]
Yes I think Maven does a basic reverse sort to show the latest.

I looked into this before and unfortunately Maven doesn't allow deletion.

That nf-execution version is very old and should not be used.
